config/default/core.entity_view_display.paragraph.hs_hero_image.hero_with_overlay.yml

```diff
@@ -0,0 +1,119 @@
+uuid: 61e018e7-05e8-4ab0-8292-7fab15057d05
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.paragraph.hero_with_overlay
+    - field.field.paragraph.hs_hero_image.field_hs_hero_body
+    - field.field.paragraph.hs_hero_image.field_hs_hero_image
+    - field.field.paragraph.hs_hero_image.field_hs_hero_link
+    - field.field.paragraph.hs_hero_image.field_hs_hero_overlay_color
+    - field.field.paragraph.hs_hero_image.field_hs_hero_title
+    - paragraphs.paragraphs_type.hs_hero_image
+  module:
+    - ds
+    - field_formatter_class
+    - hs_field_helpers
+    - layout_builder
+    - link
+    - smart_trim
+    - stanford_media
+third_party_settings:
+  layout_builder:
+    allow_custom: false
+    enabled: false
+  ds:
+    layout:
+      id: pattern_hero_text_overlay
+      library: null
+      disable_css: false
+      entity_classes: all_classes
+      settings:
+        pattern:
+          field_templates: default
+          variant: _field
+          variant_field: field_hs_hero_overlay_color
+          variant_field_values:
+            default: 00505c
```

Comment:
Why are some with quotes and others not?

Reply:
:man_shrugging: i wondered the same. thats just how it exported